charter.md

```diff
@@ -72,28 +78,156 @@ this works.
 
 ### Vacancies
 
-In the event of a resignation or other loss of an elected steering committee
-member, the candidate with the next most votes from the previous election will
-be offered the seat.  This process will continue until the seat is filled.
+In the event of a resignation or other loss of an elected committee member, the
+next most preferred candidate from the previous election will be offered the
+seat.
+
+A maximum of one (1) committee member may be selected this way between
+elections.
 
 In case this fails to fill the seat, a special election for that position will
-be held as soon as possible. [Eligible voters][voter-eligibility] from the most
-recent election will vote in the special election i.e., eligibility will not be
-redetermined at the time of the special election. A committee member elected in
-a special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder.
+be held as soon as possible.
+
+Examples (non-exhaustive):
+
+* the next most preferred candidate is no longer interested in sitting on the
+  committee
+* the next most preferred candidate cannot sit on the committee, due to some
+  committee composition requirement i.e.,
+  [maximal representation][maximal-representation]
 
+[Eligible voters][voter-eligibility] from the most recent election will vote in
+the special election i.e., eligibility will not be redetermined at the time of
+the special election.
+
+A committee member elected in a special election will serve out the remainder
+of the term for the person they are replacing, regardless of the length of that
+remainder.
+
+[maximal-representation]: /elections.md#maximal-representation
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
```

Comment:
Curious: I didn't track all the discussion here.  This is effectively "unanimous".  Are we formulating it this way because "maybe one day the committee will grow to 9 and then 3/4 will be 7/9 (which, as you know, has a certain meaning :)?

Reply:
There was a fair amount of back and forth, but we opted to follow the 3/4 vote required for charter changes

Reply:
I think we started off using this as an example to illustrate, but it just stuck there. (thinking say we lose a member to resignation and immediately after we end up with a bad situation unrelated to the other person and have to deal with it)

Reply:
> Curious: I didn't track all the discussion here. This is effectively "unanimous". Are we formulating it this way because "maybe one day the committee will grow to 9 and then 3/4 will be 7/9 (which, as you know, has a certain meaning :)?

@thockin --

1. LOL
2. What Bob [said](https://github.com/kubernetes/steering/pull/248#discussion_r917460116):
   > There was a fair amount of back and forth, but we opted to follow the 3/4 vote required for charter changes

   Based on the discussions, folks felt that a two-thirds supermajority was too few.

---

> thinking say we lose a member to resignation and immediately after we end up with a bad situation unrelated to the other person and have to deal with it

@dims -- I think this is a special case that we haven't discussed yet. Let's do so on https://github.com/kubernetes/steering/issues/250.